jsonargparse/deprecated.py

```diff
@@ -250,3 +250,8 @@ def set_url_support(enabled:bool):
     Setting the logger property to None was deprecated in v4.10.0 and will raise
     an exception in v5.0.0. Use False instead.
 """
+
+env_prefix_property_none_message = """
+    Setting the env_prefix property to None was deprecated in v4.11.0 and will raise
+    an exception in v5.0.0. Use False instead.
```

Comment:
Should be "Use True instead" since that is what will have the same behavior as before with None.

Reply:
Done